spec/services/sign_in/token_serializer_spec.rb

```diff
@@ -128,6 +128,21 @@
           expect(subject).to eq({})
         end
       end
+
+      context 'when client_config has refresh_token_path' do
+        it 'returns the refresh_token_path from client_config' do
+          subject
+          expect(cookies[refresh_token_cookie_name][:path]).to eq(refresh_path)
```

Comment:
you need to set `refresh_path` to something besides the default here. This needs to test that if the `client_config.refresh_token_path` is set to something besides the default it will return that path in the cookie.